Mathlib/LinearAlgebra/StdBasis.lean

```diff
@@ -321,7 +322,7 @@ end Module
 
 namespace Matrix
 
-variable (R : Type*) (m n : Type*) [Fintype m] [Fintype n] [Semiring R]
+variable (R : Type*) (m n : Type*) [Fintype m] [Finite n] [Semiring R]
```

Comment:
What about `m`?

Reply:
We can make it `Finite` but this requires more work. Currently it's used in a definition.